package.json

```diff
@@ -22,11 +22,16 @@
   "main": "dist/main.js",
   "peerDependencies": {
     "prop-types": "~15.5.8",
-    "react": "^15.4.2",
-    "react-dom": "^15.4.2"
+    "react": "^16.2.0 <17.0.0",
+    "react-dom": "^16.2.0 <17.0.0"
   },
   "dependencies": {
     "classnames": "^2.2.5",
+    "enzyme-adapter-react-16": "^1.1.1",
+    "npm": "^5.6.0",
+    "raf": "^3.4.0",
+    "react": "^16.2.0",
+    "react-dom": "^16.2.0",
```

Comment:
sorry for the late comment.. but I think these dependencies should not be forced unto the users:
- `react` and `react-dom` is already mentioned as peerDependencies,
- `npm` is not used, not sure why it's there in the first place
- `enzyme-adapter-react-16` is only used for development, and user does not necessarily test their code using `enzyme`
- `raf` is not the only option for polyfilling `requestAnimationFrame`.. user might already do it through other method.. and if `react` them self doesn't force one particular way to polyfill it, I don't think that we should too..

opened a PR to fix these points: https://github.com/patw0929/react-intl-tel-input/pull/199